rclcpp/src/rclcpp/executor.cpp

```diff
@@ -373,6 +373,12 @@ Executor::spin_some_impl(std::chrono::nanoseconds max_duration, bool exhaustive)
   wait_for_work(std::chrono::milliseconds(0));
   bool just_waited = true;
 
+  if(entities_need_rebuild_) {
```

Comment:
Will probably fail lint? Might just need `ament_uncrustify --reformat .`.

```suggestion
  if (entities_need_rebuild_) {
```

Reply:
Weirdly that doesn't seem to cause the linter to fail... seems fishy.

Reply:
i thought that too, this weird thing happens to rcl too? https://github.com/ros2/rcl/pull/1201/files#r1910772257

Reply:
> Will probably fail lint? Might just need `ament_uncrustify --reformat .`.

I run exactly this command, before every commit....

Reply:
Note: I've been seeing this in Nav2 recently as well. Here nor there, but the linters seem like they're missing things :upside_down_face:

Reply:
In order to move to the new uncrustify for the transition to Ubuntu 24.04, I had to relax the uncrustify rules quite a bit.  So it is possible that uncrustify now allows things that used to be caught.  I'd be on board with tightening those back up, but someone would need to spend the time looking back through the uncrustify configuration, changing those rules, and then fixing whatever fallout there is.